requirements.txt

```diff
@@ -8,11 +8,13 @@ pip
 pybind11
 pyyaml
 scipy
+pandas
```

Comment:
let's only require it when running dynamic batch. Pands should not be a common requirement.